.github/workflows/build.yml

```diff
@@ -46,33 +46,53 @@ jobs:
 
   build_unix:
     needs: prepare
-    runs-on: ubuntu-18.04  # Standalone executable should be built on minimum supported OS
+    runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-python@v4
       with:
           python-version: '3.10'
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+          miniforge-variant: Mambaforge
+          use-mamba: true
+          channels: conda-forge
+          auto-update-conda: true
+          activate-environment: ''
+          auto-activate-base: false
+
     - name: Install Requirements
       run: |
-          sudo apt-get -y install zip pandoc man
+          sudo apt-get -y install zip pandoc man sed
```

Comment:
why is `sed` added to a `apt install` when sed in preinstalled

Reply:
Actually, everything but **pandoc** is already installed. It seemed obvious to me that this is the list of packages used, for which you cannot be 100% sure that they are already installed. I'd rather add **make** here. After all, `ubuntu-latest` is a moving target. If you know the logic the existing list follows: just tell me.

Reply:
It is unlikely `sed` will ever be removed as a pre-installed software. But no harm in explicitly declaring it either